packages/grid/x-data-grid/src/components/containers/GridRootStyles.ts

```diff
@@ -128,7 +128,28 @@ export const GridRootStyles = styled('div', {
     '--unstable_DataGrid-overlayBackground': theme.vars
       ? `rgba(${theme.vars.palette.background.defaultChannel} / ${theme.vars.palette.action.disabledOpacity})`
       : alpha(theme.palette.background.default, theme.palette.action.disabledOpacity),
+    '--unstable_DataGrid-containerBackground': theme.vars
+      ? `rgba(${theme.vars.palette.background.defaultChannel} / 0.8)`
```

Comment:
if we use the space notation, then?

```suggestion
      ? `rgb(${theme.vars.palette.background.defaultChannel} / 0.8)`
```